modules/ossm-understanding-versions.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+// * service_mesh/v1x/upgrading-ossm.adoc  ???
+// * service_mesh/v2x/upgrading-ossm.adoc
+// * service_mesh/v2x/ossm-troubleshooting.adoc
+
+[id="ossm-versions_{context}"]
+= Understanding Service Mesh versioning
+
+The {ProductName} 2.0 Operator supports both v1 and v2 service meshes.
+
+* *Operator* version - The current Operator version is {ProductVersion}. This version number only indicates the version of the currently installed Operator. This version number is controlled by the intersection of the *Update Channel* and *Approval Strategy* specified in your Operator subscription. The version of the Operator does not determine which version of the `ServiceMeshControlPlane` resource is deployed. Upgrading to the latest Operator does *not* automatically upgrade your service mesh control plane to the latest version.
+
+[IMPORTANT]
+====
+Upgrading to the latest Operator version does not automatically upgrade your control plane to the latest version.
+====
```

Comment:
```suggestion
+
[IMPORTANT]
====
Upgrading to the latest Operator version does not automatically upgrade your control plane to the latest version.
====
```

Not sure if this needs a + here to make the note part of this bullet?
https://deploy-preview-29410--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-troubleshooting-istio.html#ossm-versions_troubleshooting-ossm